### ISSUE TITLE
DateField: [aria-*] attributes do not have valid values

### ISSUE THREAD

**Issue by mvysny** [vaadin/framework#11872]
Vaadin 8.9.4, Chrome 79, Tomcat

Analyzing a page with a DateField with Chrome Audits (Running just Audits / Accessibility is enough) will print the following failure:

![Screenshot from 2020-01-15 12-23-08](https://user-images.githubusercontent.com/254091/72426160-e7628300-3791-11ea-9b4d-fefcdd83f658.png)

The problem is that the `<input>` element uses the following attribute `aria-controls="gwt-uid-2"` but there is no such element with ID `gwt-uid-2`: `document.getElementById('gwt-uid-2')` returns null  in JavaScript console.

**Comment by mvysny** [comment]
The UI which reproduces the issue:
```
@Theme("mytheme")
public class MyUI extends UI {

    @Override
    protected void init(VaadinRequest vaadinRequest) {
        final VerticalLayout layout = new VerticalLayout();

        final DateField dateField = new DateField("Foo");
        layout.addComponent(dateField);

        setContent(layout);
    }

    @WebServlet(urlPatterns = "/*", name = "MyUIServlet", asyncSupported = true)
    @VaadinServletConfiguration(ui = MyUI.class, productionMode = false)
    public static class MyUIServlet extends VaadinServlet {
    }
}
```
